manifests/job.pp

```diff
@@ -34,6 +37,7 @@
   $jobname  = $title,
   $enabled  = 1,
   $ensure   = 'present',
+  $folder   = undef,
```

Comment:
If `$folder` is defined, it should be possible to add a dep along the lines of `Jenkins::Job[$folder] -> Jenkins::Job[$title]` so the user doesn't need to explicitly declare it.